README.md

```diff
@@ -4,6 +4,11 @@
 
 This repository is primarily concerned with back end infrastructure like datastores, servers, and APIs. The pipeline that feeds data into this system can be found in the [Openverse Catalog repository](https://github.com/WordPress/openverse-catalog). A front end web application that interfaces with the API can be found at the [Openverse frontend repository](https://github.com/WordPress/openverse-frontend).
 
+## Getting started
+
+Our quickstart guide and other documentation can be found [on our github.io site](https://wordpress.github.io/openverse-api/) (or within the repo at [`./api/docs/guides/quickstart.md`](./api/docs/guides/quickstart.md)).
```

Comment:
Should we link directly to the quick start page? I actually didn't know about this github.io page (very cool!) but the home page and the top level "guides" pages are currently empty or just have a 🚧 emoji on them and could be confusing.

```suggestion
Our quickstart guide and other documentation can be found [on our github.io site](https://wordpress.github.io/openverse-api/guides/quickstart.html) (or within the repo at [`./api/docs/guides/quickstart.md`](./api/docs/guides/quickstart.md)).
```

Reply:
Getting started in the `README.md` should directly point to the most relevant 'Quickstart' page.

Reply:
Yea the only reason I linked directly to the quickstart page was because we haven't fleshed out the landing pages yet for the guides. I'm also with Dhruv - I think being as specific as possible for getting contributors up and running would be best!